projects/smart-table/src/lib/services/table.factory.ts

```diff
@@ -51,6 +61,15 @@ export class TableFactory {
     return column;
   }
 
+
+  isValidDate(dateStr: string): boolean {
+    return /^\d{4}[-]\d{2}[-]\d{2}$/.test(dateStr);
+  }
+
+  isValidISODate(dateStr: string): boolean {
+    return /^\d{4}[-]\d{2}[-]\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z$/.test(dateStr);
```

Comment:
hier is het zeker en vast niet flexibel genoeg, want smart table kan met eender welke backend gecombineerd worden

Reply:
Kan ik hier mss beter gewoon een try-catch toevoegen ipv zelf proberen een regex check te doen?